### ISSUE TITLE
Bluetooth: after unpairing a device, I cannot pair it again 

### ISSUE THREAD

**Issue by pmts** [hifiberry/hifiberry-os#495]
**Describe the bug**
The first time I connect my phone to Hifiberry via bluetooth, it works correctly. However, if I remove the pair on my phone and try to pair it again, this results in the following error: "Failed to connect to HiFiBerry. The PIN or pair code is incorrect". The only way I found to restore that phone's ability to connect to the Hifiberry is to ssh to it and use the `bluetoothctl` command to manually remove the device:

```
# bluetoothctl
Agent registered
[bluetooth]# remove C8:9B:D7:08:0F:6C
[DEL] Device C8:9B:D7:08:0F:6C realme 9 Pro+
Device has been removed
[bluetooth]#
```

After this, I can pair my device again, but only once.

I reproduced this behaviour on 2 different android phones (Realme and Xiaomi) and on Windows 10, so it's not device dependent. 

**HiFiBerryOS version**
20230404

**HiFiBerry sound card**
DAC + DSP

**To Reproduce**
1. Enable bluetooth and start pairing on Hifiberry
2. Pair a device
3. Unpair the device
4. Try to pair the same device again
5. The pairing fails

**Expected behavior**
The device pairs succesfully

**Browser (if applicable)**

**Additional context**

**Comment by jonathanota** [comment]
I have experienced a similar issue with the 20230404 release on a Raspberry Pi4 with DAC2 Pro. 

In addition, when attempting to connect via my Intel Macbook Pro laptop, it asks for a pin code.

**Comment by Weidav** [comment]
I have the same issue with my Raspberry Pi4, I'm pretty sure this problem exists on every device and this needs to be addressed.
Until then there is a temporary solution:

1. connect to your Raspberry Pi via ssh
2. find all paired bluetooth devices with `bluetoothctl paired-devices` and copy the MAC-Adress of the device you are trying to reconnect.
3. remove this device with `bluetoothctl remove XX:XX:XX:XX:XX:XX` (replace the X's with your actual MAC-Adress)
4. restart the bluetooth service `service bluetooth restart` (it shouldn't be necessary but won't hurt)

**Comment by hifiberry** [comment]
@Weidav  Thank you for your workaround. 

While we will be looking into some way to address this, this is not very high on our priority list. If somebody wants to look into this, we're happy to integrate contributions.

**Comment by Weidav** [comment]
I'd like to look into this and see how far I can get. The contribution guidelines [link here](https://github.com/hifiberry/hifiberry-os/blob/master/doc/contributions.md) isn't working. Are there any guidelines I need to have a look at?

**Comment by hifiberry** [comment]
We're talking about a contribution to the GUI code - right?

In this case, base it on the "development" branch of https://github.com/hifiberry/create

**Comment by Weidav** [comment]
Yes, I think a component that displays the content from my workaround should do the job.

**Comment by RavanH** [comment]
The discussion on https://support.hifiberry.com/hc/en-us/community/posts/4410009712657-Cannot-connect-Bluetooth-to-hifiberryOS sounds related...

**Comment by hifiberry** [comment]
HiFiBerryOS has been rewritten from scratch and is now based on Debian. Reports filed against the older, Buildroot-based images cannot be mapped onto the new system. We are therefore closing the old issue tracker in bulk: every open issue is being closed, not this one in particular, and this is not a judgement on your report.

If you still see this problem on the current, Debian-based HiFiBerryOS, please open a new report with the bug report form: <https://github.com/hifiberry/hifiberry-os/issues/new?template=01-bug-report.yml>

The form asks for the output of `config-supportinfo`. That command exists on the current HiFiBerryOS; on the old Buildroot images you will not find it, which is itself a good sign that the report belongs to the old system.

Questions about setup or hardware are better placed with [HiFiBerry support](https://support.hifiberry.com).

This thread is now locked, so please use the links above rather than replying here.
